ui/component/app/view.jsx

```diff
@@ -246,6 +248,21 @@ function App(props: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [syncError, pathname]);
 
+  // @if TARGET='web'
+  useEffect(() => {
+    fetch(`${LBRY_TV_API}/internal/status`)
+      .then(response => response.json())
+      .then(status => {
+        if (status.general_state !== 'ok') {
+          throw Error();
+        }
+      })
+      .catch(err => {
+        setCurrentlyDegradedPerformance(true);
```

Comment:
Can we do this?

1. If `general_state` is `down` or API does not return a 200, show full screen gerbil error (no ability to interact)
1. If `general_state` is not `ok` or `down`, show degraded performance

This will both:
1. Improve UX when endpoint is down since if endpoint is down site is unlikely to be working at all
2. Allow Andrey/Niko to evolve their end without another update from us (return `down` when users shouldn't be interacting but their endpoint is up)